user-manuals/facilities/manage-locations.md

```diff
@@ -1,6 +1,6 @@
 # Manage Locations
 
-Effective stock management relies on proficient location management. To enable store associates to establish precise locations for managing stock, which will ease in receiving and storing inventory, as well as facilitating the picking process, ​​retailers can define specific locations within facilities, such as Aisle, Section, Level, or Position. Users can manage locations in the facility using the `Locations` tab in the footer on the `Facility details` page. 
+Effective stock management relies on proficient location management. To enable store associates to establish precise locations for managing stock, which will ease in receiving and storing inventory, as well as facilitating the picking process, ​​retailers can define specific locations within facilities, such as Area, Aisle, Section, Level, or Sequence. Users can manage locations in the facility using the `Locations` tab in the footer on the `Facility details` page. 
```

Comment:
the column is called position in the UI so that is why we have kept it as position here. I think it should be the same as the UI/app so we can ignore this change.

Reply:
I checked the UI of the facilities app from the launchpad, it is showing level and sequence currently, could there be some discrepancy or confusion?